migration-engine/migration-engine-tests/tests/diagnose_migration_history/diagnose_migration_history_tests.rs

```diff
@@ -159,7 +159,9 @@ async fn diagnose_migration_history_without_opt_in_to_shadow_database_does_not_c
     Ok(())
 }
 
-#[test_each_connector(ignore("postgres"))]
+// TODO: We originally ignored Postgres here too, and for some reason we get no
+// drift for SQL Server, even though the `SELECT YOLO;` fails...
```

Comment:
I think the reason why these two fail is that the error is caught earlier (at the parse + bind phase), so nothing is run, and there is no drift as a consequence. We would need a more clever way to make these scripts fail (like adding a unique constraint on a column with duplicate values).